src/main/java/xyz/vegaone/easytracking/domain/UserEntity.java

```diff
@@ -16,6 +16,9 @@
     @Column(name = "email")
     private String email;
 
+    @Column(name = "project_id")
```

Comment:
It just hit me. This is a ManyToMany relationship. One user can be part of many projects and one project can have many users. 
The liquibase changesets need to changed as well. Reach out to me if you need some guidance.